tests/test_transformers/test_sqltransform.py

```diff
@@ -1,4 +1,13 @@
-from collections import Counter
+"""
```

Comment:
Moved this test file as it is actually testing the RelationalModelTransformer (I will merge it with the existing RMT tests in another PR).

Reply:
Many of the tests in this file were a bit vague (testing counts rather than checking contents) or they were written in an unintuitive way, so I fixed them up after moving the file.